Assets/Plugins/Source/Editor/Build/PlatformSpecificBuilder.cs

```diff
@@ -31,10 +31,8 @@ namespace PlayEveryWare.EpicOnlineServices.Build
     using UnityEditor.Build.Reporting;
     using Debug = UnityEngine.Debug;
     using UnityEditor;
-    using PlayEveryWare.EpicOnlineServices.Editor;
     using System;
-    using System.Threading.Tasks;
-    using Utility;
+    using Config = EpicOnlineServices.Config;
```

Comment:
noncrit: Having both `using Config` and `using Config = EpicOnlineServices.Config` is odd at first sight, but apparently completely legal to do. Seems to work. PrebuildConfig depends on this using, so it is fine.